javascript/javascript1/week3/javascript.js

```diff
@@ -0,0 +1,130 @@
+// Remove name from array list
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+for (var i = names.length - 1; i >= 0; i--) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1);
+    }
+}
+console.log(names);
+
+// Travel time Information function
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+}
+function notThisFunctionName(travelInformation) {
+    const travelTime = travelInformation.destinationDistance / travelInformation.speed;
+    return travelTime;
+}
+const travelTime = notThisFunctionName(travelInformation);
+console.log(travelTime)
+
+const hour = Math.floor(travelTime);
+let decpart = travelTime - hour;
+const min = 1 / 60;
+decpart = min * Math.round(decpart / min);
+const minute = Math.floor(decpart * 60) + '';
+console.log(`${hour} hours and ${minute} minutes`)
+
+// Time Calculation of  series Duration in Life. 
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 6,
+        hours: 15,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 9,
+        hours: 3,
+        minutes: 0,
+    },
+];
+
+function logOutSeriesText() {
+    // write code here
+    let totalPerLifeSpan = 0;
+    for (let i = 0; i < seriesDurations.length; i++) {
+        const age = 80;
```

Comment:
may be ,you can define this variable before for loop , so it not need to assign in every loop.